terraform/modules/ecs/task-definitions/worker.json.tftpl

```diff
@@ -0,0 +1,32 @@
+[{
+    "logConfiguration": {
+        "logDriver": "awslogs",
+        "secretOptions": null,
+        "options": {
+            "awslogs-group": "polytomic-worker",
+            "awslogs-region": "${region}",
+            "awslogs-stream-prefix": "service",
+            "awslogs-create-group": "true"
+        }
+    },
+    "environment": [
+        %{ for key, value in env ~}
+        {
+            "name": "${key}",
+            "value": "${value}"
+        },
+        %{ endfor ~}
+        {
+            "name": "ROLE",
+            "value": "worker"
+        }
+    ],
+    "mountPoints": [
+    {
+        "containerPath": "/var/polytomic",
+        "sourceVolume": "polytomic"
+    }],
+    "memoryReservation": 512,
```

Comment:
Let's bump the memory reservation for this to 2048 since it winds up running lots of "services".